### PR TITLE
Add tests for code/shell execution & improve config fixture

### PR DESCRIPTION
**Pull request by rickythefox**
### Background
There is no integration test for code execution using docker.

### Changes
Added integration test.

### Documentation
N/A

### Test Plan
Ran the test and it passed.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: tests/integration/execute_code_test.py
Comment (Pwuts) [inline tests/integration/execute_code_test.py:1]: Please rename this to `test_execute_code.py`, and rewrite using `pytest` instead of `unittest`

**Review by Pwuts — CHANGES_REQUESTED** [review]

File: tests/integration/execute_code_test.py
Comment (rickythefox) [reply tests/integration/execute_code_test.py:1]: Done, glad to use pytest instead.

File: tests/integration/test_execute_code.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import pathlib
+import random
+import string
+import tempfile
+from unittest.mock import MagicMock
+
+import pytest
+
+import autogpt.commands.execute_code as sut
```
Comment (Pwuts) [inline tests/integration/test_execute_code.py]: sut?

Comment (rickythefox) [reply tests/integration/test_execute_code.py]: System Under Test - https://en.wikipedia.org/wiki/System_under_test . More details - https://learn.microsoft.com/en-us/archive/blogs/ploeh/naming-sut-test-variables .

**Review by Pwuts — APPROVED** [review]
Made a few additions

**Pull request merged**